crates/bevy_core_pipeline/src/taa/mod.rs

```diff
@@ -437,7 +437,9 @@ fn extract_taa_settings(mut commands: Commands, mut main_world: ResMut<MainWorld
     {
         let has_perspective_projection = matches!(camera_projection, Projection::Perspective(_));
         if camera.is_active && has_perspective_projection {
-            commands.get_or_spawn(entity).insert(taa_settings.clone());
+            commands
+                .get_or_spawn(entity)
+                .insert((taa_settings.clone(), MipBias(-1.0)));
```

Comment:
This leaves MipBias unconfigurable when TAA is enabled. I think this setting would be better expressed as a `mip_bias_override: Option<f32>` on TaaSettings that overrides the default, but only when set to some. This can then be set to a default `Some(-1.0)` value.

Reply:
The current approach would also be subject to extraction timing issues if MipBias is set manually (the bias selected would depend on what extract system runs first).

Reply:
I added an additional_mip_bias field. It wasn't too clean though, not sure if you have a better idea on how to handle it.

Reply:
Hmmm idk if adding mip biases together is ever going to be a desirable behavior?

Reply:
The system ordering complexity does seem merited though. Not much we can do about that I think.